### PR TITLE
Remove top warnings and bump version to 1.0.0

### LINKED ISSUE #209: Remove warnings about not using in production before GA release
There are multiple warnings peppered throughout the documentation noting that SecureDrop Workstation is currently in a closed, private beta, and that it should not be used in production.

In conjunction with the GA release of SecureDrop Workstation, we'll want to make sure we have removed those warnings (although it's still worth suggesting organizations contact us first before using in production).

### PR DESCRIPTION
**Pull request by nathandyer**
This PR removes the top warnings (references to the SDW Pilot) and also bumps the version number to 1.0.0.

Closes #209

### CODE REVIEW

File: docs/general/known_issues.rst [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
-For the duration of the pilot, bugs and other issues found in
-*SecureDrop Workstation* should be reported by pilot participants via the
-`support portal <https://support.freedom.press>`_. Issues that are not instance-
-specific will be added to the appropriate public issue tracker by the developers,
-for example:
+Please report sensitive issues that are specific to your instance
+via the `support portal <https://support.freedom.press>`_. 
+
+Bugs and other issues that are not specific to your instance can be reported
+via Github using the following links:
```
Comment (legoktm) [inline docs/general/known_issues.rst:11]: nit:

```suggestion
via GitHub using the following links:
```

Maybe also "If you encounter a security-related issue, please see [SECURITY.md](https://github.com/freedomofpress/securedrop-workstation/blob/main/SECURITY.md) for instructions on how to privately report it"??

**Review by legoktm — CHANGES_REQUESTED** [review]
Nice, only one thing re security issues.

File: docs/general/status.rst [hunk 2]
```diff
@@ -1,18 +1,15 @@
 SecureDrop Workstation Project Status
 =====================================
 
-SecureDrop Workstation is currently in active development, and has not yet
-had a stable release.
-
-The general availability release is tentatively planned for Q3/Q4 2024.
+SecureDrop Workstation is currently in active development.
 
 Freedom of the Press Foundation has operated a pilot program with a limited
```
Comment (legoktm) [inline docs/general/status.rst:6]:
```suggestion
Freedom of the Press Foundation had operated a pilot program with a limited
```

File: docs/general/status.rst [hunk 3]
```diff
@@ -1,18 +1,15 @@
...
 number of organizations whose environments were especially well-suited for
-early testing of the Workstation. This pilot program is now closed. No new
-pilot participants will be considered or accepted, and current pilot
-participants can expect to transition to the standard support experience
-before the general availability release is made available later in 2024.
+early testing of the Workstation. This pilot program is now closed.
```
Comment (legoktm) [inline docs/general/status.rst:8]: Maybe?

```suggestion
early testing of the Workstation. This pilot program has now ended.
```

File: docs/general/known_issues.rst (hunk 1 above)
Comment (nathandyer) [reply docs/general/known_issues.rst]: Great suggestion! Added :)

**Comment by nathandyer** [comment]
Thanks @legoktm! I believe I addressed your feedback. One note, I've never done the in-line commits before while also trying to rebase to make the commits look pretty, while still preserving the `co-authored by` information; all that is to say, if it's a mess or needs to be changed, let me know what needs to be done and I'll polish it up.

**Review by legoktm — APPROVED** [review]
:shipit:

**Pull request merged**